test/index.js

```diff
@@ -18,6 +18,8 @@ global.d      = new Date();
 Promise.longStackTraces();
 
 describe('Query Building Tests', function() {
+  this.timeout(process.env.KNEX_TEST_TIMEOUT || 5000);
```

Comment:
I have not tested this, but would we be able to kill the duplication by making this call in a global `beforeEach()` call? As I recall, mocha has a top-level implicit anonymous test suite containing everything.